### ISSUE TITLE
Checkboxes / Radios: Action cannot be performed by keyboard alone (on certain platforms)

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#790]
@VaniChinnam which platforms does this not work on?

- https://cauldron.dequelabs.com/components/Checkbox
- https://cauldron.dequelabs.com/components/DescriptionList
- https://cauldron.dequelabs.com/components/RadioGroup
- https://cauldron.dequelabs.com/components/RadioCardGroup

**Comment by scurker** [comment]
Also raised here: https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/f841d3d2-2cd6-11ed-a675-2fb045bce52a?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=17&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb

**Comment by VaniChinnam** [comment]
@schne324 & @scurker We have tested this on Windows & Chrome. Today when I investigated it further, I found the scenario where it is not working. This behavior is due to an extension.
When the auditor extension - axe Auditor 3.0.3 is enabled, we are unable to select the checkboxes in the Cauldron and when we activate the spacebar, the focus is going to the header.
This behavior is only seen in cauldron components, the checkboxes in the auditor application are working fine though.

So I believe this is not a violation of WCAG, as the default behavior is working fine. But is there a way we fix this, as this is a cauldron component that will be used across all our deque products?

@melaniephilipp & @goodwitch your inputs on this, please?
Here are the ticket details - [(https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/f841d3d2-2cd6-11ed-a675-2fb045bce52a?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=17&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb)]

**Comment by melaniephilipp** [comment]
Wow! This is very strange. I agree it shouldn't be counted against the VPAT. However, finding out why the components don't work with the Auditor extension enabled sounds really important. I wonder if other extensions could also interfere with the components. The radio buttons and checkboxes look like they are using native HTML input type=radio and type=checkbox, so I'm curious how this could even happen?

**Comment by goodwitch** [comment]
@VaniChinnam to clean up our data...I'm removing the "VPAT" label from this issue.  Let me know if you disagree.
